pkg/plugin/plugin_test.go

```diff
@@ -663,6 +664,37 @@ var testFunc = func(version string) {
 				Expect(string(output[:len(output)-1])).To(Equal(ovsOutput))
 			})
 		})
+		Context("specified OfportRequest", func() {
+			It("should configure an ovs interface with a specific ofport", func() {
+				// Pick a random ofport 5000-6000
+				ofport_request := rand.Intn(1000) + 5000
+				ovsOutput := fmt.Sprintf("ofport              : %d", ofport_request)
+
+				conf := fmt.Sprintf(`{
+				"cniVersion": "%s",
+				"name": "mynet",
+				"type": "ovs",
+				"ofport_request": %d,
+				"bridge": "%s"}`, version, ofport_request, bridgeName)
+
+				targetNs := newNS()
+				defer func() {
+					closeNS(targetNs)
+				}()
+
+				result := attach(targetNs, conf, IFNAME, "", "")
+				hostIface := result.Interfaces[0]
+
+				// A short sleep is necessary for OVS to actually assign the port,
+				// before checking ovs-vsctl, even though the add returns successfully,
+				// ofport can still be blank ("[]").
+				time.Sleep(100 * time.Millisecond)
```

Comment:
Please use polling via gomega's `Eventually` instead: https://onsi.github.io/gomega/#category-2-making-codeeventuallycode-assertions-on-functions

Reply:
Thanks for the suggestion. Addressed in 9539d20672c637b283c06f53311a54932ebadfe5